saw-remote-api/python/saw_client/solver_cache.py

```diff
@@ -0,0 +1,124 @@
+import cbor2
+from collections.abc import MutableMapping
+from dataclasses import dataclass
+from datetime import datetime
+import dateutil.parser
+import lmdb
```

Comment:
The remote API tests are failing in CI in part because the `cbor2` and `lmdb` libraries aren't declared in the `pyproject.toml` configuration.